### PR TITLE
Feature/rename test cases

### PR DESCRIPTION
**Pull request by JensHeinrich**
##### SUMMARY

Create better readability for test cases

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

certificate_complete_chain

##### ADDITIONAL INFORMATION

Aims to make creating testcases for  #355 easier

```paste below

```

### CODE REVIEW

File: tests/integration/targets/certificate_complete_chain/tasks/main.yml [hunk 1]
```diff
@@ -63,7 +68,8 @@
       - cert2_rootchain.complete_chain | join('') == (lookup('file', 'cert2.pem', rstrip=False) ~ lookup('file', 'cert2-chain.pem', rstrip=False) ~ lookup('file', 'cert2-root.pem', rstrip=False))
       - cert2_rootchain.chain[:-1] | join('') == lookup('file', 'cert2-chain.pem', rstrip=False)
       - cert2_rootchain.root == lookup('file', 'cert2-root.pem', rstrip=False)
-  - name: Find alternate rootchain for cert 2
+
+  - name: Find alternate rootchain for cert 2 using intermediate and root pem
```
Comment (felixfontein) [inline tests/integration/targets/certificate_complete_chain/tasks/main.yml:72]:
```suggestion
  - name: Find alternate rootchain for cert 2 using intermediate and root PEM
```

File: tests/integration/targets/certificate_complete_chain/tasks/main.yml [hunk 2]
```diff
@@ -49,7 +53,8 @@
       that:
       - cert2_root.complete_chain | join('') == (lookup('file', 'cert2.pem', rstrip=False) ~ lookup('file', 'cert2-chain.pem', rstrip=False) ~ lookup('file', 'cert2-root.pem', rstrip=False))
       - cert2_root.root == lookup('file', 'cert2-root.pem', rstrip=False)
-  - name: Find rootchain for cert 2
+
+  - name: Find rootchain for cert 2 using intermediate and root pem
```
Comment (felixfontein) [inline tests/integration/targets/certificate_complete_chain/tasks/main.yml:57]:
```suggestion
  - name: Find rootchain for cert 2 using intermediate and root PEM
```

File: tests/integration/targets/certificate_complete_chain/tasks/main.yml [hunk 3]
```diff
@@ -24,7 +26,8 @@
       that:
       - cert1_root.complete_chain | join('') == (lookup('file', 'cert1.pem', rstrip=False) ~ lookup('file', 'cert1-chain.pem', rstrip=False) ~ lookup('file', 'cert1-root.pem', rstrip=False))
       - cert1_root.root == lookup('file', 'cert1-root.pem', rstrip=False)
-  - name: Find rootchain for cert 1
+
+  - name: Find rootchain for cert 1 using intermediate and root pem
```
Comment (felixfontein) [inline tests/integration/targets/certificate_complete_chain/tasks/main.yml:30]:
```suggestion
  - name: Find rootchain for cert 1 using intermediate and root PEM
```

**Comment by felixfontein** [comment]
@JensHeinrich thanks for this improvement!

**Comment by JensHeinrich** [comment]
You are welcome, I am still thinking about making the test cases better to read by using multi line strings, vars, etc. before trying to create more test cases as suggested in #357

**Pull request merged**